crates/astria-composer/src/telemetry.rs

```diff
@@ -0,0 +1,56 @@
+use color_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+use is_terminal::IsTerminal as _;
+use tracing_subscriber::{
+    filter::{
+        EnvFilter,
+        LevelFilter,
+    },
+    fmt::{
+        self,
+        MakeWriter,
+    },
+    layer::SubscriberExt as _,
+    registry,
+    util::SubscriberInitExt as _,
+};
+
+/// Register a global tracing subscriber.
+///
+/// # Errors
+///
+/// Returns the same errors as [`SubscriberInitExt::try_init`] called
+/// on a [`tracing_subscriber::Registry`].
+pub fn init<S>(sink: S, filter_directives: &str) -> eyre::Result<()>
```

Comment:
pub?

Reply:
This is part of composer's public interface (called in `main.rs` and `tests/`)